backend/accounts/baseaccount.go

```diff
@@ -57,6 +57,8 @@ type AccountConfig struct {
 	SigningConfigurations signing.Configurations
 	GetNotifier           func(signing.Configurations) Notifier
 	GetSaveFilename       func(suggestedFilename string) string
+	// Opens a file in a default application. The filename is not checked.
+	UnsafeSystemOpen func(filename string) error
```

Comment:
What do you mean by "The filename is not checked." ?

Reply:
simply that there are no checks on the filename and that the caller has to be sure that it is a valid and safe path.